### PR TITLE
Reports in admin 4277

### PR DESCRIPTION
**Pull request by dmitrytrager**
According to: https://redmine.smartdata.solutions/issues/4277

users
devices

### CODE REVIEW

File: app/admin/user.rb [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
     id_column
     column :email
     column :username
-    column :current_sign_in_at
-    column :sign_in_count
-    column :created_at
+    column :device_count do |user|
+      user.devices.count
+    end
+    column :app_count do |_|
```
Comment (dmitrytrager) [inline app/admin/user.rb]: what do we mean by app_count for users @t-harps ?

Comment (t-harps) [inline app/admin/user.rb]: Number of developers they have approved. @dmitrytrager

Comment (dmitrytrager) [inline app/admin/user.rb]: Got it. Thanks, @t-harps

File: app/admin/device.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    column :developer_count do |device|
+      device.developers.count
+    end
+    column :user_count do |device|
+      device.permitted_users.count
+    end
+    column :configurer do |device|
+      configurer = device.configurer
+
+      configurer.present? ? configurer.email : "Not set"
```
Comment (t-harps) [inline app/admin/device.rb]: Once we have a Developer table would be good to link to the developer (configurer) from this column if possible.

Comment (dmitrytrager) [inline app/admin/device.rb]: Ok, will count this in future tasks.

**Pull request merged**